README.md

```diff
@@ -1,219 +1,14 @@
-# Zendesk App Boilerplate (Vuejs + ES6)
-The scope of this project is a to have a ready to go app boilerplate to start building zendesk apps using:
+# Vue.js boilerplate
 
+> Vue.js boilerplate for custom Zendesk apps
 
-* [Vuejs](https://vuejs.org/v2/guide/)
+## Build Setup
 
-* [Vuex](https://vuex.vuejs.org/)
+```bash
+# to run app locally
+$ yarn start
```

Comment:
- I don't see the reason of using yarn here
- why removing the entire documentation?